src/utilities/extendSchema.js

```diff
@@ -324,9 +327,15 @@ export function extendSchemaImpl(
     const config = type.toConfig();
     const extensions = typeExtensionsMap[config.name] ?? [];
 
+    const specifiedByUrl = [
+      config.specifiedByUrl,
+      ...extensions.map(getSpecifiedByUrl),
+    ].filter(Boolean)[0];
```

Comment:
RFC: Is this code clear enough?  Any suggestions what would be more clear?

We want to:
1. use `config.specifiedByUrl` if present
2. else, check if any of the extensions include a `GraphQLSpecifiedByDirective`, and use the `url` value from there if present.

Since a Scalar could be extended multiple times, each of which could include multiple directives (but only one `GraphQLSpecifiedByDirective`), this was the best balance I could find between being clear and not overly verbose.


**Alternative 1: With ternary and `.filter(Boolean)`**
```suggestion
    const specifiedByUrl =
      config.specifiedByUrl != null
        ? config.specifiedByUrl
        : extensions.map(getSpecifiedByUrl).filter(Boolean)[0];
```

**Alternative 2: With ternary and double call to `getSpecifiedByUrl`**
```suggestion
    const specifiedByUrl =
      config.specifiedByUrl != null
        ? config.specifiedByUrl
        : getSpecifiedByUrl(extensions.find(getSpecifiedByUrl) || {});
```

Reply:
@m14t I converted it to a simple `for` loop.